src/components/ConfirmPopup.js

```diff
@@ -0,0 +1,33 @@
+import Popup from "./Popup.js";
+export default class ConfirmPopup extends Popup {
+  constructor(popupSelector, handleFormSubmit) {
+    super({ popupSelector });
+    this._handleFormSubmit = handleFormSubmit;
+  }
+
+  setEventListeners() {
+    super.setEventListeners();
+    this._popupElement.addEventListener("submit", (evt) => {
+      evt.preventDefault();
+      this._handleFormSubmit(this._card);
+    });
+  }
+
+  open(card) {
+    this._card = card;
+    super.open();
+  }
+
+  close() {
+    super.close();
+  }
+
+  //   domDeleteCard() {
+  //     this._card.domDeleteCard();
+  //   }
+
+  setLoadingConfirm(isLoading, text) {
+    const modalButton = this._popup.querySelector(".modal__button_delete");
```

Comment:
**Could be improved**

All elements found **every time** you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `getView`) only **1 time** in order not to waste resources on searching them again and again.

`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form`, `this._likeButton`, and so on.